consistency/tool/comparison/comparison_test.json

```diff
@@ -1,33 +1,238 @@
+                                "kind": "WordGroup",
+                                "word_frequency": {"abc":2, "abb":2}
+                                },
+                             0.19999999999999996], 
+                    "apigees":[           
+                        {"id":  "apigee", 
+                        "kind": "WordGroup",
+                        "word_frequency": {"abc":2, "abb":2}
+                       },
+                        0.09090909090909094],
+                    "gcloud":[           
+                        {"id": "Gcloud",
+                        "kind": "WordGroup",
+                        "word_frequency": {"abc":2, "abb":2}
+                       },
+                        0.19999999999999996]    
                                 }
     }, 
-    "none-ids":{
-        "ids": null,
+    "unique-terms":{
+        "wordgroups": [            
+            {"id": "YouTube", 
+            "kind": "WordGroup",
+            "word_frequency": {"abc":2, "abb":2}
+           }, 
+            {"id": "PlayStore", 
+            "kind": "WordGroup",
+            "word_frequency": {"abc":2, "abb":2}
+           },
+           {"id": "Stadia",
+            "kind": "WordGroup",
+            "word_frequency": {"abc":2, "abb":2}
+           }],
+        "noisewordgroup": {
+            "id": "noise",
+            "kind": "WordGroup",
+            "word_frequency": {"some":2, "noise":2}
+        },
         "words": [ "Google", "apigees", "gcloud"],
-        "closest_word_groups": null
-    }, 
+        "expected": {
+            "Google":[{"id": "noise",
+                       "kind": "WordGroup",
+                       "word_frequency": {"some":2, "noise":2}}, 1], 
+            "apigees":[{"id": "noise",
+                        "kind": "WordGroup",
+                        "word_frequency": {"some":2, "noise":2}}, 1],
+            "gcloud":[{"id": "noise",
+                       "kind": "WordGroup",
+                       "word_frequency": {"some":2, "noise":2}}, 1]}
+    },
+    "none-wordgroups":{
+        "wordgroups": null,
+        "words": [ "Google", "apigees", "gcloud"],
+        
+        "expected":null  
+    },
     "none-words":{
-        "ids": [ "google", "apigee", "Gcloud"],
+        "wordgroups": [ "google", "apigee", "Gcloud"],
         "words": null,
-        "closest_word_groups": null
-    }, 
-    "unique-terms":{
-        "ids": [ "YouTube", "PlayStore", "Stadia"],
-        "words": [ "Google", "apigees", "gcloud"],
-        "closest_word_groups": {"Google":["UNIQUE_WORD", 1], 
-                                "apigees":["UNIQUE_WORD", 1],
-                                "gcloud":["UNIQUE_WORD", 1]    
-                                }
-    }, 
+        "expected": null
+    },  
     "both-null":{
-        "ids": null,
+        "wordgroups": null,
         "words": null,
-        "closest_word_groups": null
+        "expected": null
+    },
+
+    "report-test-with-unqiue":{
```

Comment:
Can we add a testcase with a combination of unique and existing terms? Something like follows:

```
"report-test-unqiue-existing":{
"words":["userid", "google"],
        "wordgroups": [
            {
                "id": "Userid",
                "kind":"WordGroup",
                "word_frequency":{
                    "Userid": 1,
                    "userid": 1,
                }
            }

        ], 
        "noisegroup":{
                "id": "NOISE_WORDS",
                "kind":"WordGroup",
                "word_frequency":{
                    "Google": 1
                }

        },
        "expected_report":{
            "id": "consistency-report", 
            "kind": "ConsistencyReport",
            "current_variations":[
                {
                    "term": "userid", 
                    "cluster":{
                        "id": "Userid",
                        "kind":"WordGroup",
                        "word_frequency":{
                            "Userid": 1
                            "userid": 1 
                        }
                    }

                }
            ],
            "unique_terms": ["google"]
        }
    }, 
```